### PR TITLE
Add AKS OpenAI Embeddings workshop

### PR DESCRIPTION
**Pull request by mosabami**
We ran a levelup on this workshop earlier today that was attended by over 800 people with positive feedback. Looking to add this to the catalog so others can benefit from it

### CODE REVIEW

File: workshops/aks-openai-embeddings/workshop.md [hunk 1]
```diff
@@ -0,0 +1,371 @@
+---
+published: false                        # Optional. Set to true to publish the workshop (default: false)
```
Comment (sinedied) [inline workshops/aks-openai-embeddings/workshop.md:2]: you should set this to `true` so it's searchable in the workshop list

File: workshops/aks-openai-embeddings/assets/banner.jpg
Comment (sinedied) [inline workshops/aks-openai-embeddings/assets/banner.jpg:1]: If you don't use a banner you can remove this file, it was meant as a placeholder to create you own

**Review by sinedied — CHANGES_REQUESTED** [review]
Thanks! I added 2 small comments to fix before merging

**Comment by mosabami** [comment]
thanks @sinedied . Updated accordingly

**Review by sinedied — APPROVED** [review]

**Pull request merged**